sql/create.sql

```diff
@@ -1,5 +1,7 @@
+DROP DATABASE IF EXISTS flightdb2;
 CREATE DATABASE flightdb2;
 
+DROP USER IF EXISTS openflights@localhost;
```

Comment:
Not that it set one before... passwordless is fun :).

Obviously doesn't matter much for a dev type install.

Probably should be punted somewhere else for documentation updates though. Doesn't necessarily need fixing in this commit

Reply:
I'll remove this and the drop db from the PR.